.php_cs.dist

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$finder = new PhpCsFixer\Finder();
+$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
+$finder->in(__DIR__);
+
+/* Based on ^2.1 of php-cs-fixer */
+$config
+    ->setRules([
+        // default
+        '@PSR2' => true,
```

Comment:
Mostly Symfony overrides.

- The array syntax is because of the PHP supported version.
- The concat with space seemed the most prevalent choice already in the codebase.
- No unused imports I generally turn off since the detection does not work well with use statements that are only referenced in docblocks (at least that used to be my experience).
- No useless return and ordered imports are part of contrib.
- PHP doc align is a part of Symfony but didn't seem the most prevalent approach in current codebase.
- PHP doc no package is part of Symfony and removes all @package docblocks. Not sure if you wanted to keep these so I did not remove them for now.
- Trailing comma in array is part of Symfony was not yet the most prevalent approach  so I excluded it.